infrateam_first_responder.md

```diff
@@ -40,6 +40,12 @@ The idiosyncratic name of the role is intentional. It is not the duty of the fir
 
 # Duties
 
+## Help vet systemic puppet changes
+
+_Experiment starting July 2021_
+
+Ops will give a bit of advance notice to the infrastructure team before deploying systemic puppet changes (e.g. refactor/cleanup that touches many apps, networking changes, etc).  First responder will work with ops to determine whether testing is needed, and if so, what testing is appropriate, based on what we worry might break.  Running `infrastructure-integration-tests`  against stage after the puppet changes have been deployed there is a good default practice.  Don't hesitate to ask for help planning or performing the testing if needed.
```

Comment:
I feel like this is all a little bit too undefined to put in this document yet.  
How much notice should be given? Should Andrew, Vivian and other PSMers have a role?  How will we sign off on these changes?

Reply:
While all of JCoyne's questions are valid, thinking in terms of the new hires, having something mentioned here is probably warranted.  I think the suggested text is fine.  We don't have all the details documented for a lot of what we do.